machine/accton/accton_as5916_54x/busybox/patches/series

```diff
@@ -0,0 +1 @@
+onie-syseeprom-write-2b-delay-4000.patch
```

Comment:
Agree.  Generally speaking, it needs to generate a patch for each new platform.  Fortunately, it has only a few patch up to now that sharing between platforms.  For long-term support, using busybox config to tune the write delay would be better then using a patch.  I'll create an additional patchset for the change.  Thank you.